_team/randy-hart.md

```diff
@@ -0,0 +1,17 @@
+---
+city: Washington
+first_name: Randy
+full_name: Randy Hart
+github:
+last_name: Hart
+name: randy-hart
+role: Procurement specialist
```

Comment:
Yeah, I don't even know if that's the right title. Will need Randy or someone from consulting to weigh in on that.